sequencer/worker.go

```diff
@@ -187,6 +188,15 @@ func (w *Worker) DeleteTx(txHash common.Hash, addr common.Address) {
 func (w *Worker) UpdateTx(txHash common.Hash, addr common.Address, counters state.ZKCounters) {
 	w.workerMutex.Lock()
 	defer w.workerMutex.Unlock()
+	log.Infof("UpdateTx tx(%s) addr(%s)", txHash.String(), addr.String())
+	log.Infof("UpdateTx counters.CumulativeGasUsed: %s", counters.CumulativeGasUsed)
```

Comment:
as I said, we can set this counters logs as Debug level if you want (and left only the log in line 191 as Info level)